source/jsonnet/england-wales/household/blocks/who-lives-here/people_who_live_here_introduction.jsonnet

```diff
@@ -23,6 +23,19 @@ local contentDescription = {
       {
         description: contentDescription,
       },
+      {
+        definition: {
+          title: 'What we mean by “household”',
```

Comment:
Just a minor thing, twisty_ header should be `What we mean by "this household"`.